nixos/lib/make-system-tarball.sh

```diff
@@ -1,5 +1,4 @@
 source $stdenv/setup
-set -x
```

Comment:
Why?

Reply:
it was overly verbose, and i thought that this is typically only kept in when debugging things